### DISCUSSION TITLE
MMJ get shadow!!

### DISCUSSION THREAD

**Discussion by nassau-tk** [weihuoya/citra#758] [General]
20230331 has shadow!!

https://user-images.githubusercontent.com/48179091/229383459-a640bc5c-cd12-472d-b584-9dbdf9f3ed2e.mp4

**Comment by Tappy-tech** [comment]
Too late for that. Everybody know about that already.

**Reply by nassau-tk** [reply]
Hi,Teacher!
Please tell me that is this appear on this ver. right?
